.eslintrc.js

```diff
@@ -8,23 +8,13 @@ module.exports = {
     "eslint:recommended",
     "plugin:react/recommended",
     "plugin:@typescript-eslint/recommended",
-    "prettier",
+    // "prettier",
```

Comment:
I have to pull this out for now because pre-commit ci tries to fix eslint errors, so it would start applying prettier changes before we were ready. I created an issue so we don't forget to add it here: https://github.com/button-inc/digital_marketplace/issues/38